data/INRA/2016_sensory_analysis_essay_ALFColomb.ttl

```diff
@@ -0,0 +1,129 @@
+ 
+<data/winemaking/PechRouge/sensoryAnalysis/2016> a qb:DataSet; 
+  rdfs:label "Data about different wines which have been tested after fermentation at Pech Rouge in 2016"; 
+  qb:structure <DSD/sensoryAnalysis/2016> . 
+ 
+bdg:sensorialAnalysis a qb:AttributeProperty, qb:CodedProperty; 
+  qb:codeList <sensorialAnalysis>; 
```

Comment:
I think between "sensorialAnalysis" and "sensoryAnalysis" some poor data guy down the line will lose his hair :0 So let's find a better name for hte coded values, "Flavor" or "TastingFlavor" seems good to me, what do you think?